test/configure-slot-test.js

```diff
@@ -1,57 +1,36 @@
-var webdriver = require('selenium-webdriver'),
-    By = webdriver.By,
-    until = webdriver.until,
-    chrome = require('selenium-webdriver/chrome'),
-    nw = require('nw'),
-    path = require('path'),
-    chai = require('chai'),
-    chaiAsPromised = require('chai-as-promised'),
-    expect = chai.expect;
+const webdriver = require('selenium-webdriver');
+const By = webdriver.By;
+const until = webdriver.until;
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const expect = chai.expect;
+
+const driver = require('./driver.js');
 
 chai.use(chaiAsPromised);
 
 describe('Configuring a slot on the OnlyKey', function() {
 
-    var driver;
-
-    before(function() {
-        // Use chromedriver that comes with nwjs
-        var service = new chrome.ServiceBuilder(path.join(path.dirname(require.resolve('nw')), 'nwjs', 'chromedriver')).build();
-        chrome.setDefaultService(service);
-
-        // Point chromedriver to the nwjs app
-        var options = new chrome.Options()
-            .addArguments('nwapp=' + path.dirname(__dirname));
-
-        driver = new webdriver.Builder()
-            .withCapabilities(webdriver.Capabilities.chrome())
-            .setChromeOptions(options)
-            .build();
-    });
-
-    after(function() {
-        //return driver.quit();
-    });
-
     function expectDialogOpen(id) {
-        var dialog = driver.findElement(By.id(id));
+        let dialog = driver.findElement(By.id(id));
```

Comment:
Harmless question: why not `const` in all these tests? When would you want to allow `dialog` to be reassigned?